### ISSUE TITLE
Error seeing snps on JBrowse

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#1768]
I am getting an error message for all the files in my snp analysis.  The screenshot is attached.
<img width="1422" alt="screen shot 2017-11-02 at 11 45 37 am" src="https://user-images.githubusercontent.com/11475316/32335626-a52ae9ac-bfc3-11e7-80db-d4ac4221f20e.png">

**Comment by ARWattam** [comment]
The bam file is not loading
![screen shot 2017-11-02 at 12 24 22 pm](https://user-images.githubusercontent.com/11475316/32337649-f8008614-bfc8-11e7-90ca-3d1f92133072.png)

**Comment by ARWattam** [comment]
Also, I know that a snp should be right here, but that is not loading, either.
![screen shot 2017-11-02 at 12 24 42 pm](https://user-images.githubusercontent.com/11475316/32337681-0c9bfe3c-bfc9-11e7-88c1-44e428800bf3.png)

**Comment by mshukla1** [comment]
I think you need to select respective VCF tracks using the check boxes in the left panel.

**Comment by mshukla1** [comment]
@ARWattam, I believe this worked for you later, right? 

Please comment if you still see the problem. 

-Maulik

**Comment by hyoo** [comment]
@ARWattam Would you confirm this issue? If this still exists, would you give us process or show to @zreyn so that he can produce and debug? If this is your job result, then, maybe you can share this job result with him.

**Comment by rkenyon** [comment]
Seems to be working for the test run I did.

![screen shot 2018-08-16 at 2 56 00 pm](https://user-images.githubusercontent.com/3188570/44228968-a6ada700-a164-11e8-82cd-0bb37300b05e.png)

**Comment by rkenyon** [comment]
vcf.gz track never showed anything, but no errors.
![screen shot 2018-08-16 at 3 02 27 pm](https://user-images.githubusercontent.com/3188570/44229249-77e40080-a165-11e8-8173-8b39927d5869.png)

**Comment by ARWattam** [comment]
Yes, I see them all now, and have even confirmed that the data in the html matches what I see in the browser.  Excellent job and well done.
